tox.ini

```diff
@@ -0,0 +1,34 @@
+[tox]
+envlist =
+    py{26,27,32,34,35,36,37,38,39,310}
```

Comment:
do we need/want `py26`?

Reply:
I've added `py26` because we still support Python 2.6 https://github.com/slashmili/python-jalali/blob/f7af4724cd47ecccf6e8cda8520b41213ce71ff3/setup.py#L23

Btw, I think we don't need to support Python2.6 anymore. I removed `py26` from `tox.ini`. I will create a PR later to remove Python2.r support.